### ISSUE TITLE
Move repo

### ISSUE THREAD

**Issue by dougiesquire** [aidanheerdegen/nchash#6]
As with [yamanifest](https://github.com/aidanheerdegen/yamanifest/issues/13), shall we move to https://github.com/ACCESS-NRI?

**Comment by aidanheerdegen** [comment]
Yes, though I'm unsure if it should be ACCESS-NRI or ACCESS-Hive, reflecting the diverse background of the tool. Might be a nice example to show how community projects could be re-homed there as well.

**Comment by aidanheerdegen** [comment]
Also bear in mind that this isn't used, and maybe should be removed as a dependency. It was an idea to make a fast netcdf change detection hash, but I ended up just using `binhash`
